components/Nav/Nav.tsx

```diff
@@ -28,6 +32,16 @@ export const Nav = (): JSX.Element => {
   const { logoutAccount } = useAuthContext();
   const [open, setOpen] = React.useState(false);
 
+  const [leagues, setLeagues] = useState<ILeague[]>([]);
+  const { user } = useDataStore((state) => state);
+
+  useEffect(() => {
+    if (!user.id || user.id === '') {
+      return;
+    }
+    setLeagues(user.leagues);
+  }, [user]);
+
   /**
```

Comment:
We need to fetch the user leagues from another call rather than pulling it in from the user object.  See `/leagues/all` for how we are getting all user leagues.